htdocs/commande/card.php

```diff
@@ -1730,32 +1731,32 @@
 	// Delivery delay
 	print '<tr class="fielddeliverydelay"><td>'.$langs->trans('AvailabilityPeriod').'</td><td>';
 	print img_picto('', 'clock', 'class="pictofixedwidth"');
-	$form->selectAvailabilityDelay($availability_id, 'availability_id', '', 1, 'maxwidth200 widthcentpercentminusx');
+	$form->selectAvailabilityDelay((GETPOST('availability_id')?GETPOST('availability_id'):$availability_id), 'availability_id', '', 1, 'maxwidth200 widthcentpercentminusx');
```

Comment:
Use
`GETPOSTISSET('availability_id')?GETPOST('availability_id'):$availability_id`
instead of
`GETPOST('availability_id')?GETPOST('availability_id'):$availability_id`